src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Application/GivenVacancySearchParametersMapper/AndPostedInLastNumberOfDays.cs

```diff
@@ -10,23 +10,21 @@ namespace Esfa.Vacancy.Register.UnitTests.SearchApprenticeship.Application.Given
     [TestFixture]
     public class AndPostedInLastNumberOfDays
     {
-        private List<string> _expectedStandards = new List<string> { "STDSEC.9", "STDSEC.3", "STDSEC.8" };
-        [Test]
-        public async Task ThenReturnFromDateAccordingly()
-        {
-            var result = VacancySearchParametersMapper.Convert(new SearchApprenticeshipVacanciesRequest()
-            { PostedInLastNumberOfDays = 2, StandardCodes = _expectedStandards });
+        private readonly List<string> _standardCodes = new List<string> { "9" };
 
-            result.FromDate.Should().Be(DateTime.Today.AddDays(-2), "From date is that many days ahead from today");
-        }
-
-        [Test]
-        public async Task ThenReturnNullFromDate()
+        [TestCase(2, TestName = "Then from date should be two days ago")]
+        [TestCase(null, TestName = "Then from date should be null")]
+        [TestCase(0, TestName = "Then from date should be today")]
+        public async Task ThenPopulateFromDateAccordingly(int? sinceDays)
```

Comment:
I recommend modifying the names of the tests:

the test func name would be something like WhenPopulatingFromDate or similar
and the test cases would have test names of "And PostedInLastNumberOfDays is null then from date should be null" and so on for the other test cases. That way when someone else is modifying the code and gets a test failure we know exactly why.